torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10470,6 +10485,18 @@ def error_inputs_mean(op_info, device, **kwargs):
             ErrorInput(si2, error_regex=err_msg2),
             ErrorInput(si3, error_regex=err_msg3))
 
+def reference_flatten(input, start_dim=0, end_dim=-1):
```

Comment:
Add a brief comment describing this function

Reply:
I'm sorry how is this resolved? I still don't see a comment describing this function

Reply:
Sorry, I was sure I added that comment but you are right. It is there now.